tests/test.nf

```diff
@@ -21,10 +19,27 @@ process RNG {
 
 }
 
+// shows nf-prov behavior with exec tasks
+process EXEC_FOO {
+    tag "${prefix}"
+    publishDir "${params.outdir}", mode: 'copy'
+
+    input:
+    tuple val(prefix), val(constant)
+
+    output:
+    path(outputfile), emit: "txt"
```

Comment:
```suggestion
    path("${prefix}.exec_foo.txt"), emit: "txt"
```

I think you need to do this for the output file to be picked up correctly

Reply:
did not work for me, no difference

Reply:
is the file present in the task directory?

Reply:
yes